Editorial proposed changes for cc section (#3491)
quicwg/base-drafts: draft-ietf-quic-recovery.md

## Congestion Avoidance

Slow start exits to congestion avoidance.  Congestion avoidance uses an
Additive Increase Multiplicative Decrease (AIMD) approach that increases
the congestion window by one maximum packet size per congestion window
acknowledged.  When a loss or ECN-CE marking is detected, NewReno halves
the congestion window, sets the slow start threshold to the new
congestion window, and then enters the recovery period.

## Recovery Period

A recovery period is entered when loss or ECN-CE marking of a packet is

> detected in congestion aviodance but after the congestion wiondow and slow

Reviewer: ```suggestion
detected in congestion avoidance but after the congestion window and slow
```

> start threhold has been adapted.  A recovery period ends when a packet sent

Reviewer: ```suggestion
start threshold has been adapted.  A recovery period ends when a packet sent
```

> start threhold has been adapted.  A recovery period ends when a packet sent

Reviewer: adapted how?

Reviewer 2: Addressed in #3505, PTAL.

## Pacing {#pacing}

> This document does not specify a pacer, but it is RECOMMENDED that a sender paces

Reviewer: ```suggestion
This document does not specify a pacer, but it is RECOMMENDED that a sender pace
```
